Controller/PageController.php

```diff
@@ -52,7 +52,7 @@ public function exceptionsListAction()
      *
      * @return Response
      */
-    public function exceptionEditAction($code)
+    public function exceptionEditAction(Request $request, $code)
```

Comment:
This is a BC-break

Reply:
Why is it a BC break? Symfony will resolve the typehinted parameter and the `$code` parameter will be mathed to the route parameter

Reply:
> Or do you mean that the controller can possibly be extended?

Yes

Reply:
Or should I change this to `public function exceptionEditAction($code, Request $request = null)` and when this is null use the old method to get the request + deprecration when null? This way you can remove the BC method in the next major release and make the request parameter required

Reply:
Nope: https://3v4l.org/Wmd05

Reply:
oh snap indeed :stuck_out_tongue:

Reply:
Fixed!